functions-and-scope.js

```diff
@@ -73,7 +128,17 @@ for (let i = 0; i < 8; i++){
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
-
+function checkHighestGrade(value){
+    let highestGrade = 0;
```

Comment:
Het is beter om als default waarde altijd het eerste cijfer van de array te pakken. Cijfers zullen in dit voorbeeld altijd hoger dan 0 zijn, maar voer je functie eens uit met een array als `[-1, -2, -4, -1]`. Dan zal er `0` uitkomen terwijl die waarde niet eens in de array voorkomt 😉